contracts/script/multisigTransactionProposals/safeSDK/modifyProverModeProposal.ts

```diff
@@ -28,17 +32,24 @@ async function main() {
     const safeAddress = getEnvVar("SAFE_MULTISIG_ADDRESS");
     isValidEthereumAddress(safeAddress);
     const safeSdk = await Safe.create({ ethAdapter, safeAddress });
-
-    const permissionedProverAddress = getEnvVar("APPROVED_PROVER_ADDRESS");
-    isValidEthereumAddress(permissionedProverAddress);
-
-    await proposeSetProverTransaction(
-      safeSdk,
-      safeService,
-      await orchestratorSigner.getAddress(),
-      safeAddress,
-      permissionedProverAddress,
-    );
+    const orchestratorSignerAddress = await orchestratorSigner.getAddress();
+
+    if (command == SET_PROVER_CMD) {
+      console.log(`${command}`);
+      const permissionedProverAddress = getEnvVar("APPROVED_PROVER_ADDRESS");
+      isValidEthereumAddress(permissionedProverAddress);
```

Comment:
I think the name `isValidEthereumAddress` doesn't really convey that this function throws.  By the name alone, it seems like this would just return a `boolean`, and it looks like this `boolean` isn't inspected at all which seems odd, and makes the function call seem pointless.

However, upon inspecting the function, it seems that this function will return `true` or throw an `Error`.  So it seems like this is really a validation function for the address to ensure that it is a correctly formatted address.  I think returning `true` in this case is unecessary as nothing is even done with the result.  I this this function should probably be named more appropriately to reflect that this is validating.

There's also room for improvement with the error message returned here, to provide the context of which address is throwing the error.

Reply:
Good point, i've updated it here https://github.com/EspressoSystems/espresso-sequencer/commit/418c16ff56e0a0ad830c4fd0f47bf2c5c941a9c6